packages/client/.env.example

```diff
@@ -1,2 +1,3 @@
-REACT_APP_FLOW_ENV=emulator
-REACT_APP_FLOWGRAPH_API_KEY=REACT_APP_FLOWGRAPH_API_KEY
+REACT_APP_FLOW_ENV=emulator
+REACT_APP_GRAFFLE_COMPANY_ID=f64c4980-ccea-45c2-9c9c-ffc160c5eca4

```

Comment:
Should these IDs be public?

Reply:
Theses are ids are not sensitive and we want people consuming the indexer so it's included in .env.example